server.py

```diff
@@ -0,0 +1,114 @@
+from flask import Flask,jsonify, request
+import os
+import shutil
+import pathlib
+
+
+app = Flask(__name__)
+
+#==============================================heartbeat<3=========================
+
+@app.route('/heartbeat' , methods=['GET', 'POST'])
+def heartbeat():
+    return "204"
+
+#==============================================reading file=========================
+@app.route('/GET/<string:pathToFile>', methods=['GET'])
+def GetFile(pathToFile):
+    if os.path.exists(pathToFile):
```

Comment:
It should read files from the storage directory, not simply from the root (will explain in details later)